TripPlanner/TripPlanner.Controllers/Controllers/ViewBusinessTripsController.cs

```diff
@@ -31,5 +32,15 @@ public async Task<IEnumerable<IBusinessTrip>> GetTrips([FromBody] SearchCriteria
            
             return trips;
         }
+
+        [Route("UpdateTrip")]
+        [HttpPut]
```

Comment:
Use query param for status instead body request
Use id in the url for the trip

Reply:
Done